### ISSUE TITLE
Display CGM Data in mmol vs. mg/dL

### ISSUE THREAD

**Issue by apabari** [loudnate/Loop#87]
Was able to pull some CGM data from Dexcom Share.  As a Canadian user I am displaying data in mmol; however, data is only being displayed in mg/DL in loop.  

Could this be a configurable item for the CGM data graph?

**Comment by loudnate** [comment]
Absolutely, though no setting is necessary because the setting exists in HealthKit. 
https://github.com/loudnate/LoopKit/issues/27 is a pre-requisite here. 

Please don't hesitate to file any additional issues you encounter with units.

**Comment by apabari** [comment]
Is a code change required to enable this?  My settings in HealthKit are in mmol.  Interestingly, the Glucose value listed in loop (at the bottom of the app) is displayed in mmol.  But the graph view is displaying in mg/DL.

**Comment by loudnate** [comment]
Yes, this is something I plan to fix. If you'd like to just quickly change your code in the meantime, you'll find two "TODO" items in this file. 

https://github.com/loudnate/Loop/blob/master/Loop/Managers/StatusChartManager.swift

**Comment by apabari** [comment]
I just tried modifying the code, but I noticed that the glucose predictions are not being displayed properly when I change the units.  The graph is also showing from 0 to 50 units, which makes the line almost flat regardless. I'm not sure how to change that -- my own lack of knowledge of this software language, I'm sure.  If you have a chance to direct me at which lines I need to modify, in the meantime, that would be great.

Thanks!
